challenge.rb

```diff
@@ -1,26 +1,27 @@
 
-def capitalize_each_string(input)
+def capitalize_each_string(*animal)
   #implement your solution here
+  animal.each { |a| puts a.capitalize }
```

Comment:
Run  your entire file by typing "ruby challenge.rb". You should see "True" output six times on six separate lines if you've done everything correctly.